docs/components/organization/create-organization.mdx

```diff
@@ -71,9 +73,248 @@ All props below are optional.
+
+#### `mountCreateOrganization()` usage
+
+<InjectKeys>
+
+<CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
+  ```js filename="main.js" {14}
+  import Clerk from '@clerk/clerk-js';
+
+  // Initialize Clerk with your Clerk publishable key
+  const clerk = new Clerk('{{pub_key}}');
+  await clerk.load();
+
+  document.getElementById("app").innerHTML = `
+    <div id="create-organization"></div>
+  `;
+
+  const createOrgDiv =
+    document.getElementById("create-organization");
+
+  clerk.mountCreateOrganization(createOrgDiv);
+  ```
+
+  ```html filename="index.html" {11}
+  <!-- Add a <div id="create-organization"> element to your HTML -->
+  <div id="create-organization"></div>
+
+  <script>
+    window.addEventListener("load", async function () {
+      await Clerk.load();
+
+      const createOrgDiv =
+        document.getElementById("create-organization");
+
+      Clerk.mountCreateOrganization(createOrgDiv);
+    });
+  </script>
+  ```
+</CodeBlockTabs>
+
+</InjectKeys>
+
+### `unmountCreateOrganization()`
+
+Unmount and run cleanup on an existing `<CreateOrganization />` component instance.
+
+```typescript
+function unmountCreateOrganization(node: HTMLDivElement): void;
+```
+
+#### `unmountCreateOrganization()` params
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `node` | [`HTMLDivElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement) | The container `<div>` element with a rendered `<CreateOrganization />` component instance |
+
+#### `unmountCreateOrganization()` usage
+
+<InjectKeys>
+
+<CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
+  ```js filename="main.js" {18}
+  import Clerk from '@clerk/clerk-js';
+
+  // Initialize Clerk with your Clerk publishable key
+  const clerk = new Clerk('{{pub_key}}');
+  await clerk.load();
+
+  document.getElementById('app').innerHTML = `
+    <div id="create-organization"></div>
+  `
+
+  const createOrgDiv =
+    document.getElementById('create-organization');
+
+  clerk.mountCreateOrganization(createOrgDiv);
+
+  // ...
+
+  clerk.unmountCreateOrganization(createOrgDiv);
+  ```
+
+  ```html filename="index.html" {15}
+  <!-- Add a <div id="create-organization"> element to your HTML -->
+  <div id="create-organization"></div>
+
+  <script>
+    window.addEventListener("load", async function () {
+      await Clerk.load();
+
+      const createOrgDiv =
+        document.getElementById('create-organization');
+
+      Clerk.mountCreateOrganization(createOrgDiv);
 
-[components-ref]: /docs/components/overview
-[ap-ref]: /docs/account-portal/overview
+      // ...
+
+      Clerk.unmountCreateOrganization(createOrgDiv);
+    });
+  </script>
+  ```
+</CodeBlockTabs>
+
+</InjectKeys>
+
+### `openCreateOrganization()`
+
+Opens the `<CreateOrganization />` component as an overlay at the root of your HTML `body` element.
```

Comment:
Not sure what this line means, is it possible to make it a little more explicit? From my reading, it seems to add something to the `body` element?

Reply:
yes the component is opened as an overlay. like a popup